docs/labwc.1.scd

```diff
@@ -16,6 +16,14 @@ It is light-weight and independent with a focus on simply stacking windows
 well and rendering some window decorations. Where practicable it uses clients
 for wall-paper, panels, screenshots and so on.
 
+Exit and reconfigure the compositor on SIGTERM and SIGHUP respectively.
```

Comment:
I get what this means, but I stumble a bit over the wording. A couple of alternatives:

- The compositor will exit or reload its configuration upon receiving SIGTERM or SIGHUP respectively.
- One can exit or reconfigure the compositor by sending it SIGTERM or SIGHUP respectively.

Reply:
Thanks. I've used the first suggestions. That's much better English.